applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Intakes/ApplicantResult.cs

```diff
@@ -4,31 +4,24 @@ namespace Unity.GrantManager.Intakes
 {
     public class ApplicantResult
     {
-        [JsonPropertyName("id")]
```

Comment:
I feel like there was a specific reason that we had explicit json property names for these fields - just double checking that its not relevant anymore

Reply:
It is returning fine without it